Update registry.json

krypt01337 opened the pull request:
### Commitment
<!-- A brief overview of the commitment you are making by joining the CSPA. -->

By submitting this PR I agree that I will fight for the full decentralization of Cardano by only running a single pool and reporting members that are breaking the rules by running multiple pools.

#### PR Checklist
<!-- A list of items to check before raising your PR. -->

I have:
- [x] Added a telegram handle of an **individual** and not that of a group or channel.
- [x] Added a social media account with activity.
- [x] Removed any `@` characters from Telegram, Instagram or Twitter handles.
- [x] Provided my operator name as my real full name, my real first name, or I have left that field blank.
- [x] Included all social media fields in the PR even if they are blank (this will fail validation if not done).

krypt01337 commented:
new SPO registration

Bez625 commented on registry.json:
```diff
@@ -5419,5 +5419,19 @@
+   "ticker": "1337",
+   "poolId": "a84d7b46aef878ae8873710bdb9ee3119094dc0bbe0d80e94f53cb9e",
+   "social": {
+     "twitter": "",
```
There is no social media presence for this pool. Please can you add your twitter or other so we can see how you communicate with your delegates and attract new ones? This is just so we can verify you are not part of a pool group.

krypt01337 commented:
Hi.I have added my Telegram account.Is that enough?http://1337.chiplife.art/metadata/1337extended.jsonBest RegardsTobbe --Sent from my Android phone with GMX Mail. Please excuse my brevity.On 2022-08-13, 13:18 Chris Berry ***@***.***> wrote:

   
  @Bez625 requested changes on this pull request. 
   
  In registry.json: 
  > @@ -5419,5 +5419,19 @@
    "telegram": "Dinoaus73",
    "operatorName": "John Dean",
    "github": ""
+  },
+  {
+   "ticker": "1337",
+   "poolId": "a84d7b46aef878ae8873710bdb9ee3119094dc0bbe0d80e94f53cb9e",
+   "social": {
+     "twitter": "",
 
  There is no social media presence for this pool. Please can you add your twitter or other so we can see how you communicate with your delegates and attract new ones? This is just so we can verify you are not part of a pool group. 
  —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

Bez625 commented:
> Hi.I have added my Telegram account.Is that enough?

No you must also add a social media presence. This is so we can verify that the pool is indeed a single pool and not part of a group

Bez625 commented:
Since there is no update I will close this PR. Please feel free to raise another if you want to join the alliance.

krypt01337 commented:
Hi again. I added  social media presence right after my last message. But I must have forgot to report it. 
Do I have do start this process all over now?
I have my Discord handle in the .json file:
http://1337.chiplife.art/metadata/1337extended.json

krypt01337 commented:
Ok. I have now added my Discord handle to the file:http://1337.chiplife.art/metadata/1337extended.json/Tobbe, SPO (1337)--Sent from my Android phone with GMX Mail. Please excuse my brevity.On 2022-08-26, 09:50 Chris Berry ***@***.***> wrote:

   
   
   Hi.I have added my Telegram account.Is that enough? 
   
  No you must also add a social media presence. This is so we can verify that the pool is indeed a single pool and not part of a group 
  —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
